_posts/2019-08-28-snowplow-javascript-tracker-2.11.0-released-with-gdpr-context/index.md

```diff
@@ -0,0 +1,102 @@
+
+NB. This release introduces a change to how assets are hosted - rather than hosting the tracker on cloudfront, we now publish the asset to the GitHub release. Users who have previously relied on the cloudfront hosted asset must host the tracker on their own CDN, as is recommended practice. More detail in the [New deployment process](#deployment) and [Upgrading](#upgrade) sections.
+
+Read on below the fold for:
+
+1. [Tracking GDPR basis for processing with the GDPR context](#gdpr-context)
+2. [New deployment process](#deployment)
+3. [Updates and bug fixes](#updates)
+4. [Upgrading](#upgrade)
+5. [Documentation and help](#doc)
+
+<!--more-->
+
+<h2 id="gdpr-context">1. Tracking GDPR basis for processing with the GDPR context</h2>
+
+This release introduces the `enableGdprContext` method, which once enabled appends a GDPR context to all events. This allows users to easily record the basis for data collection and relevant documentation, and enables a straightforward audit flow for all events.
+
+`enableGdprContext` takes four arguments:
+
+- GdprBasis (required string enum, must be one of the following values consent, contract, legalObligation, vitalInterests, publicTask, legitimateInterests)
+- documentId (string)
+- documentVersion (string, max 16 charas)
+- documentDescription (string)
+
+It is called as follows:
+
+{% highlight javascript %}
+window.snowplow('enableGdprContext',
+  'consent',
+  'abc1234',
+  '0.1.0',
+  'this document describes consent basis for processing'
+);
+{% endhighlight %}
+
+Documentation can be found [here][gdpr-tracker-docs]
+
+
+<h2 id="deployment">2. New deployment process</h2>
+
+Previously to this release, the javascript tracker was published to a public cloudfront asset which could be called from anywhere - this made it simpler for new users to get started with snowplow tracking. However, this represents an inherent security risk if the hosted asset is called directly in third party web pages/applications. For this reason we have always recommended that users host the tracker on their own CDN for production usage. With this release we have decided to establish a new process which enforces this as policy.
+
+The tracker asset is now published as a [Github Release][2.11.0-tag] (the `sp.js` file) - which can be hosted on your own CDN - we recommend that it is hosted on the same domain in which the tracker is called. Snowplow Insights customers will have the asset deployed to the CDN within their AWS or GCP account by default, but we encourage self-management of the process also for those interested in more control.
+
+There are additional advantages to doing this:
+
+- Upgrading tracker versions can now more easily be managed by rotating the hosted asset rather than changing client-side tracking code.
```

Comment:
```suggestion
- Upgrading tracker versions can now be managed more easily by rotating the hosted asset rather than changing client-side tracking code.
```